pkg/test/cobra.go

```diff
@@ -33,6 +33,7 @@ func ExecuteCommandC(c *cobra.Command, args ...string) (*cobra.Command, string,
 	buf := new(bytes.Buffer)
 	c.SetOutput(buf)
 	c.SetArgs(args)
+	c.SilenceUsage = true
```

Comment:
Adding this since `tkn` already has `SilenceUsage` set to true in [`root.go`](https://github.com/tektoncd/cli/blob/604f6cf20f2a7fad2ecbe523ac57b0ea4826b2ab/pkg/cmd/root.go#L71). Tests should be the same output.